pkg/services/alerting/notifiers/googlechat.go

```diff
@@ -55,6 +55,7 @@ Structs used to build a custom Google Hangouts Chat message card.
 See: https://developers.google.com/hangouts/chat/reference/message-formats/cards
 */
 type outerStruct struct {
+	FallbackText string `json:"title"`
```

Comment:
Think this needs to be ``FallbackText string `json:"fallbackText"` `` to be compatible with [hangout chat api](https://developers.google.com/hangouts/chat/reference/rest/v1/spaces.messages).

Reply:
Thank you, I don't understood exactly the third element of these kind of lines because I am very newbie with Go.